Rookies/README.md

```diff
@@ -196,6 +196,17 @@ You can read the [Heroes Review Criteria and Scoring Matrix Guidelines](../Heroe
 ### Suraj Vijayan
 <img src="./images/suraj.jpg" width="100"/>
 
+### Tugwa Chinenye Victor
```

Comment:
Hey @Vickysnipe,

Can you please make sure to fix this and ensure that your profile addition is separated out from Suraj's? Right now you have inserted your profile right in the midst of theirs.

It should be in alphabetical order by first name, so yours should come after the last "S" first name profile.